pkgs/servers/search/zinc/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildGo117Module
```

Comment:
Please don't add new packages that depend on 1.17, it is EOL and will be removed as soon as is practical.

Reply:
While the package manually builds fine with Go 1.18 (and upstream recommends it), it fails on nix (I guess because go.mod declares 1.16). Tried patching it but no luck. Closing this then I guess.